docs/layouts/parts/custom-slots-example.md

```
@@ -0,0 +1,25 @@
+<script setup lang="ts">
```

Markdownlint

Element: script
Inline HTML

(MD033, no-inline-html)